whylog/constraints/__init__.py

```diff
@@ -0,0 +1,36 @@
+class Constraint(object):
+    def __init__(self):
+        pass
+
+
+class TimeConstraint(Constraint):
+    def __init__(self, abstract_line_earlier, abstract_line_later, min_delta=None, max_delta=None):
```

Comment:
Usually arguments don't have an "abstract_" prefix. Is this constructor different?